lib/command.ts

```diff
@@ -427,7 +427,20 @@ Command.setReplyTransformer("hgetall", function (result) {
   if (Array.isArray(result)) {
     const obj = {};
```

Comment:
Could probably just use `Object.create(null)` here to avoid the performance hit from the additional check below.